README.md

```diff
@@ -1 +1,25 @@
-# Hello1-Microverse
\ No newline at end of file
+
+# Hello1-Microverse
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
+## Prerequisites
+Text editor,
+browser like crome.
+if you have the above mentioned  prerequistes you can clone the project by using the link(https://github.com/haftamudesta/portfolio) to your text editor and run it on your local machine.
+## Author
+  [Github](https://github.com/solog0039)
+  [linkedin](www.linkedin.com/in/solomon-kidanu-62a994232)
+  [tiwtter](https://twitter.com/Solomon57320119)
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+## Show your support
+give tumbs up if you like this project.
+## Acknowledgments
+Microverse
+## License
+This project is Apache License licensed.
```

Comment:
- [ ] Please as requested by the previous reviewer, use [this README template](https://github.com/microverseinc/readme-template). Using a professional README template makes your repo professional.